microsoft/iot-cardboard-js
# Visual rules flighting

### Summary of changes 🔍 
Initial pull request for refactor of Status and Alerts UI. This pull request includes a temporary lightweight flighting solution to be able to render Visual rules in the left panel. This pull request does not contain all scaffolding and type definition, that is still a work in progress.

### Testing 🧪
- Add `cardboard.feature.visualRulesPivot` key and set to true in localStorage.
- Go to Behavior form and see the pivot change.

### Checklist ✔️
